nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/InvokeScriptedProcessor.java

```diff
@@ -92,11 +92,10 @@
                     logger.error(message, t);
                 }
             }
-        } else {
-            // Return defaults for now
-            relationships.add(REL_SUCCESS);
-            relationships.add(REL_FAILURE);
         }
+        // Add defaults
+        relationships.add(REL_SUCCESS);
+        relationships.add(REL_FAILURE);
```

Comment:
@pvillard31 Please check the use of session.transfer(flowFile, InvokeScriptedProcessor.REL_SUCCESS) in test_reader.groovy#onTrigger. All groovy test cases uses REL_SUCCESS or REL_FAILURE . It is not a problem in pre-NIFI-1152 and that's why I had to raise NIFI-1838.